haiku-apps/flyingtroll/flyingtroll-1.0~git.recipe

```diff
@@ -32,12 +33,12 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	make OBJ_DIR=objects
+	make -C src OBJ_DIR=objects
 }
 
 INSTALL()
 {
 	install -d $appsDir
-	install -t $appsDir objects/FlyingTroll
-	addAppDeskbarSymlink $appsDir/AliveJournal
+	install -t $appsDir src/objects/FlyingTroll
+	addAppDeskbarSymlink $appsDir/FlyingTroll
```

Comment:
> There weren't any errors with the build, so why isn't the app loading its images?

I think this might help:
```diff
-	install -d $appsDir
+	install -d $appsDir/FlyingTroll
-	install -t $appsDir src/objects/FlyingTroll
+	install -t $appsDir/FlyingTroll src/objects/FlyingTroll
+	install -t $appsDir/FlyingTroll/images src/images/*
-	addAppDeskbarSymlink $appsDir/FlyingTroll
+	addAppDeskbarSymlink $appsDir/FlyingTroll/FlyingTroll
```
(I haven't tested, though)

Reply:
Leave the testing to me, @fbrosson  :)

Reply:
Hmm @fbrosson it doesn't seem to work in x86_64 :(

Reply:
I'll try to test x86_gcc2 also, but I have a feeling I'm going to get the same result :/

Reply:
It works for me, on x86_64 and I see the images.
Are you sure you have `install -d $appsDir/FlyingTroll/images`? Because my comment was missing the "/images" (until I fixed it).

Reply:
Wait. Let me check...

Reply:
@fbrosson do you know how to get rid of packages installed in Apps folder? I noticed that when I installed the newer hpkg the new app didn't replace the old one.

Reply:
Oh wait, never mind :)

Reply:
@fbrosson I got the images :)

Reply:
Hmm, I guess you already understood why your second installation failed :)

BTW, after a failed attempt to install a package, pkgman leaves a `transaction-1` (or -2 or -3 ...) directory in `/system/packages/administrative/` with a copy of the hpkgs you tried to install.
When that happens you can delete that `transaction-1` directory (or rerun the install from there after you uninstall the old hpkg.)